diff --git a/CMakeLists.txt b/CMakeLists.txt
index 48a1869..647d454 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,6 +17,8 @@ file(GLOB_RECURSE SRC_FILES
     "${SRC_DIR}/**.cpp"    
 )
 
+add_compile_options("/utf-8")
+
 add_library(${PROJECT_NAME} MODULE ${SRC_FILES})
 set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 23)
 
diff --git a/src/game/frontend/Menu.cpp b/src/game/frontend/Menu.cpp
index 55f5e9a..3819ebe 100644
--- a/src/game/frontend/Menu.cpp
+++ b/src/game/frontend/Menu.cpp
@@ -113,14 +113,14 @@ namespace YimMenu
 		ImFontConfig FontCfg{};
 		FontCfg.FontDataOwnedByAtlas = false;
 
-		auto font = io.Fonts->AddFontFromMemoryTTF(const_cast<std::uint8_t*>(Fonts::MainFont), sizeof(Fonts::MainFont), size, &FontCfg, io.Fonts->GetGlyphRangesDefault());
+		auto font = io.Fonts->AddFontFromMemoryTTF(const_cast<std::uint8_t*>(Fonts::MainFont), sizeof(Fonts::MainFont), size, &FontCfg, io.Fonts->GetGlyphRangesChineseFull());
 
 		// just use Arial for Cyrillic
 
-		FontCfg.MergeMode = true;
-		io.Fonts->AddFontFromFileTTF((std::filesystem::path(std::getenv("SYSTEMROOT")) / "Fonts" / "arial.ttf").string().c_str(), size, &FontCfg, GetGlyphRangesCyrillicOnly());
+		//FontCfg.MergeMode = true;
+		//io.Fonts->AddFontFromFileTTF((std::filesystem::path(std::getenv("SYSTEMROOT")) / "Fonts" / "arial.ttf").string().c_str(), size, &FontCfg, GetGlyphRangesCyrillicOnly());
 
-		io.Fonts->Build();
+		//io.Fonts->Build();
 
 		return font;
 	} 
diff --git a/src/game/frontend/fonts/Fonts.hpp b/src/game/frontend/fonts/Fonts.hpp
index dc0f801..4b67f48 100644
--- a/src/game/frontend/fonts/Fonts.hpp
+++ b/src/game/frontend/fonts/Fonts.hpp
@@ -2,6 +2,6 @@
 
 namespace YimMenu::Fonts
 {
-	extern const uint8_t MainFont[78948];
+	extern const uint8_t MainFont[1894471];
 	extern const uint8_t IconFont[1049188];
 }
\ No newline at end of file
diff --git a/src/main.cpp b/src/main.cpp
index 0bc4008..15ca166 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -80,7 +80,7 @@ namespace YimMenu
 		if (!Pointers.LateInit())
 			LOG(WARNING) << "Socialclub patterns failed to load";
 
-		Notifications::Show("YimMenuV2", "Loaded succesfully", NotificationType::Success);
+		Notifications::Show("大助手汉化", "加载成功", NotificationType::Success);
 
 		while (g_Running)
 		{
